packages/api/models/granules.js

```diff
@@ -348,60 +362,56 @@ class Granule extends Manager {
     granule,
     message,
     executionUrl,
-    executionDescription = {},
+    processingTimeInfo = {},
   }) {
     if (!granule.granuleId) throw new CumulusModelError(`Could not create granule record, invalid granuleId: ${granule.granuleId}`);
     const collectionId = getCollectionIdFromMessage(message);
     if (!collectionId) {
       throw new CumulusModelError('meta.collection required to generate a granule record');
     }
+
+    const {
+      files,
```

Comment:
This is probably not a great question, particularly given we're just going to eliminate this code eventually, and falls squarely in the 'nit' category..... but.... 

Should we consider writing a translation for the output of what we're generating in `generateGranuleRecord`/`generateFileRecord` in the sf-event lambda, instead of generating the full record here?

Reply:
🤔 So translate from the Postgres format to the API/Dynamo format? I'm not opposed, but is the idea that it saves us from having to map the cumulus message to each database schema?

Reply:
After considering this, I'm inclined to leave it as is since the Dynamo logic is ultimately going away. At worst, we could make this change in a follow up PR if we feel strongly about it

Reply:
That seems acceptable - as noted, it's a nit and I don't want this to slow progress.